test/helpers/platform-routes.js

```diff
@@ -198,6 +198,8 @@ export function createPlatformRoutes(server, opts = {}) {
 					if (user) {
 						users.push({
 							...user,
+							name: `${user.firstname} ${user.lastname}`.trim(),
+							teams: data.teams.filter(t => t.users.find(u => u.guid === user.guid)).length,
```

Comment:
Maybe we can name it "teamCount" like in other places to be more consistent?

Reply:
The `teamCount` property is only used for the `axway org view` command which calls `sdk.org.find()`. The property is no longer relevant since the PR changed `sdk.org.find()` to return all of the org's teams in addition to the count rendering the `userCount` irrelevant, but we need to keep it since removing it would be a breaking change.

Specifically referring to this instance though, this `platform-routes.js` is apart of the mocked server side and the server implementation does not return a `teamCount` property, thus our mocked test shouldn't either.